File: docs/code-examples/point2d_simple.py
"""Log some very simple points."""
import rerun as rr

rr.init("rerun_example_points2d", spawn=True)

rr.log("points", rr.Points2D([[0, 0], [1, 1]]))

# Log an extra rect to set the view bounds
rr.log("bounds", rr.Boxes2D(half_sizes=[2, 1.5]))

File: docs/code-examples/point2d_simple.rs
//! Log some very simple points.

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let (rec, storage) = rerun::RecordingStreamBuilder::new("rerun_example_points2d").memory()?;

    rec.log("points", &rerun::Points2D::new([(0.0, 0.0), (1.0, 1.0)]))?;

    // Log an extra rect to set the view bounds
    rec.log("bounds", &rerun::Boxes2D::from_half_sizes([(2.0, 1.5)]))?;

    rerun::native_viewer::show(storage.take())?;
    Ok(())
}
